README.md

```diff
@@ -72,25 +81,34 @@ vagrant ssh -- -t 'rhcd --log-level trace \
 4. You should see logs in rhcd and file with stdout of your script uploaded to the minio storage
     - Go to <http://localhost:9990/login> and use credentials from `.env` file
 
-### Bash script example
+### Worker playbooks
+
+There is an [example playbook](
+https://github.com/oamg/rhc-worker-bash/blob/main/development/nginx/data/example.yaml)
+available under `development/nginx/data`, with a minimal bash script to use
+during the worker execution.
+
+If there's a need to test any other playbook provided in this repository, one
+must change what playbook will be used during the message consumption in the
+[mqtt_publish.py](https://github.com/oamg/rhc-worker-bash/blob/main/development/python/mqtt_publish.py#L22)
+file with the name that corresponds the ones present in `development/nginx/data`. Currently, the ones available are:
+
+1. [example.yaml](https://github.com/oamg/rhc-worker-bash/blob/main/development/nginx/data/example.yaml)
+2. [convert2rhel.yaml](https://github.com/oamg/rhc-worker-bash/blob/main/development/nginx/data/convert2rhel.yaml)
```

Comment:
And this playbook will only be added in future right? Because I don't see it in this PR

Reply:
Oops, good catch! I think it got out of the commit because of gitignore. I will force add it in the next commit.

Reply:
done!